src/node/delay.rs

```diff
@@ -30,6 +30,15 @@ impl Default for DelayOptions {
     }
 }
 
+#[derive(Copy, Clone)]
```

Comment:
deriving `Default` would do nicely to prevent the large constructor down below

Reply:
of course, done!

Reply:
FYI if the default only contains zeroes you can do this:
https://github.com/orottier/web-audio-api-rs/commit/fcbce99b56d6eb8c1bb8b812312d51e89f06c532

Reply:
More simple indeed :) thanks !